frontend/styles/MainLayout.module.scss

```
@@ -1,3 +1,5 @@
+@import 'Variables.module.scss';
```

Stylelint (17.14.1)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)